play-media-native/features/ContentItemFields/ContentItemFields.tsx

```diff
@@ -135,6 +135,7 @@ export const ContentItemFields = ({
           required={overrides[fieldKey].required}
           title={overrides[fieldKey].title}
           value={state[stateKey][fieldKey]}
+          containerStyle={styles.inputContainer}
         />
       ),
       [FIELD_TYPES.Date]: (fieldKey: string) => (
```

Comment:
Title of this field should be just "Date".

Currently on Add event details, Edit event details screens it is "Time & Date"
![image](https://user-images.githubusercontent.com/96472119/226670681-3bf29b5a-3c11-475a-9e11-c7642de6f282.png)